signer/tests/integration/rbf.rs

```diff
@@ -188,7 +192,7 @@ pub fn transaction_with_rbf(
         })
         .collect();
 
-    faucet.generate_blocks(1);
+    let block_hash = faucet.generate_blocks(1);
```

Comment:
nit: I would add a `.pop().unwrap()` here, since it makes reading line 260 a little easier.

Reply:
Updated in [9b38cb1](https://github.com/stacks-network/sbtc/pull/810/commits/9b38cb1d33633cbc3655b95e53caa443d6e9ed16)